pycwt/wavelet.py

```diff
@@ -604,7 +593,8 @@ def wct_significance(al1, al2, dt, dj, s0, J, significance_level=0.95,
 
     nbins = 1000
     wlc = np.ma.zeros([J + 1, nbins])
-    for i in range(mc_count):
+    # Displays progress bar with tqdm
+    for i in tqdm.tqdm(range(mc_count)):
```

Comment:
You can change this to `` tqdm.tqdm(range(mc_count), disable=not progress)`` to give users the option to disable the progress bar.